portfolio/src/main/webapp/request.js

```diff
@@ -64,9 +65,46 @@ function onClickStartScheduling() {
  * Updates the UI to show the results of a query.
  */
 function handleScheduledTaskArray(scheduledTaskArray) {
-  const resultElement = document.getElementById('scheduled-task-list');
+  const resultElement = document.getElementById('schedule-result-list');
   resultElement.innerHTML = '';
-  scheduledTaskArray.forEach(addScheduledTaskToDom, resultElement);
+  scheduledTaskArray.forEach(addScheduledTaskToDom);
+}
+
+/**
+ * Returns a JSON of the latest scheduled tasks pulled from the DOM.
+ */
+function collectAllScheduledTasks() {
+  const allScheduledTasks = new Array();
+  // Get scheduled task list element to run through cards collecting data.
+  const scheduledTaskList = document.getElementById('schedule-result-list');
+
+  scheduledTaskList.childNodes.forEach((scheduledTaskCard) => {
+    // The cardBody of the scheduledTaskCard
+    const cardBody = scheduledTaskCard.childNodes[0];
+
+    const scheduledTaskName = cardBody.childNodes[0].innerText;
+    const scheduledTaskScheduledTime =
+        cardBody.childNodes[1].innerText.substring(
+            SCHEDULED_TIME_INITIAL_TEXT.length);
+    const scheduledTaskDurationMinutes =
+        parseInt(cardBody.childNodes[2].innerText.substring(
+            DURATION_INITIAL_TEXT.length));
+    const scheduledTaskDescription = cardBody.childNodes[3].innerText.substring(
+        DESCRIPTION_INITIAL_TEXT.length);
+    const scheduledTaskPriority =
+        parseInt(cardBody.childNodes[4].innerText.substring(
+            PRIORITY_INITIAL_TEXT.length));
+
+    var scheduledTask = {};
+    const task = new Task(
+        scheduledTaskName, scheduledTaskDescription,
```

Comment:
Consider placing one param per line. For example, when there are many params (like 7), often we end up looking for the 5th param. It's easier to find that when there is one param per line.

Reply:
Done.